lessons/RLoops/lesson.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: For Loops in R
+visible: true
+---
+<!-- change visible to true if you want it on the site -->
+
+# High Performance Loops in R
+
+ - **Authors**: Lina Tran adapted from Roz Dakin
+ - **Research field**: Neuroscience
+ - **Lesson topic**: For Loops 
+ - **Lesson content URL**: <https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/RLoops>
+
+To For Loop or not to For Loop? Learn different ways to make use of for loops as well as when it might be more efficient to use other methods in favour of looping. We will also learn how to use the Microbenchmark package to time how long it takes to execute code when deciding to use one method over another to maximize efficiency.
+
+## Topics Covered:
+
+- When loops are useful
+- When other alternatives might work better
+- The Apply family
+- How to time your code
+- Speeding up your loops
+
+##What You Will Need:
+
+- [R](https://cran.rstudio.com/)
+- [RStudio](https://www.rstudio.com/products/rstudio/#Desktop)
+
+Once you have these set up, make sure to install the following packages (gapminder, dplyr, & microbenchmark) either through Tools > Install Packages in RStudio, or in the console:
+
+```
+install.packages('gapminder')
+install.packages('dplyr')
+install.packages('microbenchmark')
+```
+
+[**Here is the lesson material with the R output**](https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/RLoops/HighPerformanceLoops.html)
```

Comment:
Almost there. The http link you provided won't work on the site as it will direct it to the github repo *file* rather than the website *page*. You have to include `(../HighPerformanceLoops.html)` for it to work on the site.

Thanks!